network/p2p/p2p.go

```diff
@@ -251,6 +258,12 @@ func (n *p2pNetwork) isReady() bool {
 func (n *p2pNetwork) UpdateSubnets(logger *zap.Logger) {
 	// TODO: this is a temporary fix to update subnets when validators are added/removed,
 	// there is a pending PR to replace this: https://github.com/ssvlabs/ssv/pull/990
+	var (
+		topicScoreUpdateInterval = 32 * n.cfg.Network.SlotDurationSec()
+		topicScoreUpdateTicker   = time.NewTicker(topicScoreUpdateInterval)
+	)
+	defer topicScoreUpdateTicker.Stop()
+
```

Comment:
I believe this was going to be used for updating the topic score parameters. But there is already a function for it (the one below this one) and these variables are unused, so we can remove it, right?

Reply:
yeah lets remove

Reply:
Removing